### ISSUE TITLE
dot plot and size circumference under Summary

### ISSUE THREAD

**Issue by deniseOme** [opentargets/genetics#193]
In the variant page:

https://genetics-app.netlify.com/variant/1_154426264_C_T

I can see the quantile when hovering over the dots under DHS-promoter 

I expected to have the same behaviour allowed for the dots under Summary? Or do we just want to give the qualitative feel to the difference in the circumference (largest: more evidence, hence more confidence -->)? 

Same applies for this locus view:
https://genetics-app.netlify.com/locus?chromosome=1&end=155426264&selectedGenes=ENSG00000163263&selectedIndexVariants=1_154426264_C_T&selectedStudies=GCST001226&start=153426264

the same would apply. Note that in the variant view users can explore the different columns to relate the quantile to the size of the dot, but in the locus example above there is no information to compare with as it's a singleton (under Overall G2V).

**Comment by edm1** [comment]
Yes the idea is to give a "qualitative feel" in the overview tab. In the per-source tabs, mouseover is viewing the underlaying data.

**Comment by ElaineMcA** [comment]
@deniseOme - do docs need a revision or clarification?

**Comment by deniseOme** [comment]
I'm not sure if this is captured in the docs. Need to check.

**Comment by ElaineMcA** [comment]
@edm1,  per meeting today, can you please update with final decision on showing the V2G overall score (normalised or raw) versus size circumference and list what specific changes (if any) are required to be actioned.

**Comment by edm1** [comment]
The current score used to determine dot size is not normalised per variant, and so would be difficult for the user to interpret. I would say we should leave it as it is now (show dot but not the underlying score).

In a future release, when we have revamped the scoring, it may make more sense to display the score. For example, the classification probability or a normalised V2G score which is more easily interpreted.
